packages/shared/lib/core/utils/interfaces/key-value-box-list.interface.ts

```diff
@@ -0,0 +1,9 @@
+export interface IKeyValueBoxList {
+    [key: string]: {
+        data: string
+        isCopyable?: boolean
+        copyValue?: string
+        isTooltipVisible?: boolean
+        alternateKey?: string
```

Comment:
Are we missing `tooltipText?: string` here? (from `TokenActivityInformation.svelte`)